backend/src/main/java/ch/ase21/backend/service/SaleService.java

```diff
@@ -248,6 +248,8 @@ private SaleService() {/* void */}
 
     if(neighbourhoodScore == null){
       neighbourhoodScore = 0.5;
+    } else {
+      neighbourhoodScore = neighbourhoodScore / 10.0;
```

Comment:
Why do you divide by 10 here if it's not null? (just for understanding)

Reply:
because I need the score to be between 0 and 1 for the overall score calculation